lang/cs/Org.Apache.REEF.Client/CLRBridgeClient.cs

```diff
@@ -17,130 +17,12 @@
-
-        public static HashSet<string> AppDll
-        {
-            get
-            {
-                HashSet<string> appDlls = new HashSet<string>();
-                appDlls.Add(typeof(HelloStartHandler).Assembly.GetName().Name);
-                appDlls.Add(typeof(HelloTask).Assembly.GetName().Name);
-                appDlls.Add(typeof(INameServer).Assembly.GetName().Name);
-                return appDlls;
-            }
-        }
-
-        public static IConfiguration ClrConfigurations
-        {
-            get
-            {
-                return DriverBridgeConfiguration.ConfigurationModule
-                .Set(DriverBridgeConfiguration.OnDriverStarted, GenericType<HelloStartHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnEvaluatorAllocated, GenericType<HelloAllocatedEvaluatorHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnEvaluatorAllocated, GenericType<AnotherHelloAllocatedEvaluatorHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnContextActive, GenericType<HelloActiveContextHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnTaskMessage, GenericType<HelloTaskMessageHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnEvaluatorFailed, GenericType<HelloFailedEvaluatorHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnTaskFailed, GenericType<HelloFailedTaskHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnTaskRunning, GenericType<HelloRunningTaskHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnEvaluatorRequested, GenericType<HelloEvaluatorRequestorHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnHttpEvent, GenericType<HelloHttpHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnEvaluatorCompleted, GenericType<HelloCompletedEvaluatorHandler>.Class)
-                .Set(DriverBridgeConfiguration.CustomTraceListeners, GenericType<DefaultCustomTraceListener>.Class)
-                .Set(DriverBridgeConfiguration.CustomTraceLevel, Level.Info.ToString())
-                .Set(DriverBridgeConfiguration.CommandLineArguments, _submitControlForAllocatedEvaluator) // this is for testing purpose only
-                .Set(DriverBridgeConfiguration.OnDriverRestarted, GenericType<HelloRestartHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnDriverReconnect, GenericType<DefaultLocalHttpDriverConnection>.Class)
-                .Set(DriverBridgeConfiguration.OnDirverRestartContextActive, GenericType<HelloDriverRestartActiveContextHandler>.Class)
-                .Set(DriverBridgeConfiguration.OnDriverRestartTaskRunning, GenericType<HelloDriverRestartRunningTaskHandler>.Class)
-                .Build();
-            }
-        }
-
+    { 
         public static void Main(string[] args)
         {
-            Console.WriteLine("start running client: " + DateTime.Now);
```

Comment:
The Main() has to stay because the project is still console application. Will remove it when changing the exe to dll in REEF-473.